nservicebus/security/encryption.md

```diff
@@ -47,8 +52,15 @@ Property encryption is enabled via the configuration API.
 
 <!-- import EncryptionServiceSimple -->
 
+### Generating encryption keys
+
+In order to use the RijndaelEncryptionService a primary and secondary keys need to be generated. An option is to use `RijndaelManaged.GenerateKey`, please refer to the following article to generate new keys:
+
+- https://msdn.microsoft.com/en-us/library/system.security.cryptography.rijndaelmanaged.generatekey(v=vs.110).aspx
```

Comment:
Don't need to generate primary and secondary. The secondary keys are for transitioning from an expired key to a new one, since even if you could change all configs in the system at once (doubtful) there still may be in-flight messages.

Also pointing to GenerateKey is not helpful. That generates a byte array. We read keys as ASCII, but you can't convert the generated key to ASCII because it will have unprintable characters. If you Base64 encode the byte array, then you get a string that's too long, so you need to take the first (can't remember how many) characters.